opg-php-fpm-71-ppa-1604/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: build
+
+registryUrl ?= registry.service.opg.digital
+
+image = opg-php71-fpm-1604
```

Comment:
this name needs to match the directory name for the master makefile to work correctly